### ISSUE TITLE
[BUG] Dynamic bones interactions working, but not viewable

### ISSUE THREAD

**Issue by wball22** [emmVRC/emmVRC-Public#1293]
Hi,
I am finding that for whatever reason, the Dynamic bones settings are only partially functioning for myself. As examples, using the rexouium avatar by rezillo ryker (has great bones for testing this with, very visible movements) we have found

1. I can interact with other people's ears and feathers/hair, verified they could see it in a mirror. I could not see any changes
2. They can interact with my ears and feathers, and see the interaction directly. I could not see it in a mirror

![bones](https://user-images.githubusercontent.com/75411265/100974013-96728f00-3509-11eb-9427-8b40a955c3ec.png)
![bones2](https://user-images.githubusercontent.com/75411265/100974022-98d4e900-3509-11eb-9cb3-04ef448c89a5.png)

Above is the current configuration emmVRC has set, we have attempted to toggle between "Friend Global Dynamic Bones" and "Everybody Global Dynamic Bones" and the same thing applies. They can see my interactions, and interact with me successfully, I see no change.

I am currently being affected by #111. I do not know if that part of the functionality is needed for this to work, but assume it is not else no interactions would work at all.

Further, I have managed to get emmVRC to throw errors occasionally, but have not been able to figure out how i produced them to reproduce them. See attached : 
[MelonLoader_20-12-02_23-29-40.632.log](https://github.com/emmVRC/emmVRC-Public/files/5634190/MelonLoader_20-12-02_23-29-40.632.log)

Finally, to test that other functionality was working, I have been able to save emmVRC favorite avatars, enable risky, and toggle ESP, Speed, and other functions. It seems that ONLY being able to see dynamic bone stuff is having issues.

**Comment by Provini** [comment]
Have you turned off "Limit Dynamic Bones" in Safety > Performance Options?

**Comment by wball22** [comment]
![bones3](https://user-images.githubusercontent.com/75411265/100976797-0aaf3180-350e-11eb-85eb-eaa8927c9102.png)

I did not. I suddenly feel this is a case of RTFM but while reading though the manual the first time I did not see anything on the dynamic bones sliders at all, so I am either blind, in which case my bad, or the manual just does not have it in which case the manuals bad (the linked documentation wiki in discord only talks about the player tweaks remove bones button), or it actually is broke. I just checked my friends list and everyone is offline. Will test tomorrow when ones i know have it enabled too are on and close this if it was me being dumb. 

Thanks for the quick reply and sorry about the possible time waste in advance.
